.github/workflows/label-pr.yml

```diff
@@ -0,0 +1,19 @@
+name: Pull Request Labels
+on:
+  pull_request:
+    types: [opened, labeled, unlabeled, synchronize, reopened]
+jobs:
+  label:
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+      pull-requests: write
+    steps:
+      - name: Label PR
```

Comment:
Can we name this step `Validate PR Labels?` or something similar? The name is a bit misleading as is.